universal-application-tool-0.0.1/app/models/Application.java

```diff
@@ -11,9 +11,20 @@
 import play.data.validation.Constraints;
 import services.applicant.ApplicantData;
 
+/**
+ * An EBean mapped class that records the submission of a single applicant to a single program at a
+ * single version.
```

Comment:
Thoughts (non-blocking): Conceptually this is true, an application is a snapshot of applicant data in time, so there is a specific version when the snapshot is taken. However, given that an application is created as draft first before it is submitted, there is nothing preventing an application from spanning different versions between the period the draft is created and when the final application is submitted as long as the program is active between versions.

Also there is no direct record/linking between an application and a version today. I've considered adding it in one of my recent PRs but gave up the idea since it doesn't seem to add much value while adding a lot unnecessary complexity.

Reply:
All I am saying is, maybe remove `at a single version`. 😃

Reply:
@syntapy yup

@hmhsu done